File: qualifier-week-3/non graded assignments/PPA7 - Not Graded.c
#include <stdio.h>
#include <math.h>

int main() {
    double radius, height, vol;
    double PI = 3.14;

    scanf("%lf", &radius);
    scanf("%lf", &height);

    vol = PI * pow(radius, 2) * height;

    printf("%.2lf", vol);
    return 0;
}

File: qualifier-week-3/non graded assignments/PPA7 - Not Graded.rs
use std::f64::consts::PI;

fn main() {
    let mut radius: f64 = 0.0;
    let mut height: f64 = 0.0;

    let _ = scanf!("{:f64}", &mut radius);
    let _ = scanf!("{:f64}", &mut height);

    println!("{:.2f}", PI * radius.powi(2) * height);
}
